Index: app/src/main/java/study/ct/com/musicapk/MainActivity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/study/ct/com/musicapk/MainActivity.java	(date 1484919446000)
+++ app/src/main/java/study/ct/com/musicapk/MainActivity.java	(revision )
@@ -9,5 +9,7 @@
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+
+
     }
 }
